README.md

```diff
@@ -66,141 +66,27 @@ at https://www.azureiotsuite.com.
 Getting Started
 ===============
 
-## Prerequisites
-* [Azure Subscription](https://azure.microsoft.com/free/) (also see [permissions guidelines](https://docs.microsoft.com/azure/iot-suite/iot-suite-permissions))
-
-## Setup
-1. Open a terminal window or command shell
-1. Check out the repository and submodules
-    ```
-    git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-java
-    ```
-    If you cloned the repo without the `--recursive` flag and it's your first time updating the submodules, you can use
-    `git submodule update --init --recursive`. More information on submodules [here](https://git-scm.com/book/en/v2/Git-Tools-    Submodules). 
-    
-1. Set up command line interface for deployments
-
-     #### Prerequisite
-    * Install [npm](https://www.npmjs.com/get-npm)
-
-    #### Mac/Linux
-    
-    ```
-    cd azure-iot-pcs-remote-monitoring-java/cli
-    npm install
-    npm start
-    npm link
-    ```
-    #### Windows
-    
-    ```
-    cd azure-iot-pcs-remote-monitoring-java\cli
-    npm install
-    npm start
-    npm link
-    ```
-    
-1. Sign into your Azure account with
-    ```
-    pcs login
-    ```
-
-## Deploy a remote monitoring solution
-### Standard Deployment
-A standard deployment is a production-ready solution for reliability and scale. *See [Basic vs. Standard Deployments](#basic-vs-standard-deployments) for more details.*
-```
-pcs -s standard --runtime java
-```
-
-### Basic Deployment
-A lower-cost showcase solution best for discovering and learning about what a Remote Monitoring preconfigured solution has to offer.  *See [Basic vs. Standard Deployments](#basic-vs-standard-deployments) for more details.*
-```
-pcs --runtime java
-```
-
-### Local Deployment
-To set up a Remote Monitoring Solution locally see
-[Local Development](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#local-development)
```

Comment:
same consideration. I think we should keep the GitHub READMEs oriented to developers, providing all the steps to work directly witht he code